tests/benchmarks/latency_measure/testcase.yaml

```diff
@@ -2,5 +2,14 @@ tests:
   benchmark.kernel.latency:
     arch_whitelist: x86 arm posix
     platform_exclude: qemu_x86_64
-    filter: CONFIG_PRINTK
+    filter: CONFIG_PRINTK and not CONFIG_SOC_FAMILY_STM32
     tags: benchmark
+
+# Cortex-M has 24bit systick, so default 1 TICK per seconds
```

Comment:
If valid for all Cortex-M, maybe it could be extended to all these.
@ioannisg what do you think ?